iot/api/README.md

```diff
@@ -57,6 +64,71 @@ service account JSON from the Google Cloud Console:
       help                  Display more information on a specific command.
       version               Display version information.
     ```
+# .NET Cloud IoT Core MQTT example
+
+This sample app publishes data to Cloud Pub/Sub using the MQTT bridge provided
+as part of Google Cloud IoT Core.
+
+Note that before you can run the sample, you must configure a Google Cloud
+PubSub topic for Cloud IoT Core and register a device as described in the
+[parent README](../README.md).
+
+## MQTT Example Setup
+
+1. From the [Google Cloud IoT Core section](https://console.cloud.google.com/iot/)
+   of the Google Cloud console, create a device registry.
+2. Use the [`generate_keys.ps1`](generate_keys.ps1) script to generate your signing keys:
+
+```powershell
+    ./generate_keys.ps1
+```
+
+3. Add a device using the file `rsa_cert.pem`, specifying RS256_X509 and using the
+  text copy of the public key starting with the ----START---- block of the certificate.
+
+```powershell
+    cat rsa_cert.pem
+```
+
+4. Connect a device using the HTTP or MQTT device samples in the [CloudMqttExample](./CloudMqttExample) folder.
+
+5. Programmattically control device configuration and using the device manager sample in the [CloudMqttExample](./CloudMqttExample) folder.
+
+## Running the sample
+
+The following command summarizes the sample usage:
+
+From a Powershell command line, restore the NuGet packages and call
+`run` to see a list of subcommands like getRegistry or getDevice:
+
+    ```powershell
+    PS C:\...\dotnet-docs-samples\iot\api\CloudIotMqttExample> dotnet restore
+    PS C:\...\dotnet-docs-samples\iot\api\CloudIotMqttExample> dotnet run
+    ```
+
+For example, if your project ID is `blue-jet-123`, your device registry is
+located in the `asia-east1` region, number of messages you want to publish is `100`,
+expiration time on your token is `15 min`, and you have generated your
+credentials using the [`generate_keys.ps1`](./generate_keys.ps1) script
+provided in the parent folder, you can run the sample as:
+
+    ```powershell
+    PS C:\...\dotnet-docs-samples\iot\api\CloudIotMqttExample> dotnet restore
+    PS C:\...\dotnet-docs-samples\iot\api\CloudIotMqttExample> dotnet run startMqtt blue-jet-123 asia-east1 device-registry-id device-id ../rsa_private.pem RS256 ../roots.pem 100 event mqtt.googleapis.com 8883 15
+    ```
+
+## Reading the messages written by the sample client
+
+1. Create a subscription to your topic.
+
+    gcloud beta pubsub subscriptions create \

```

Comment:
This one doc uses 2 formats for shell scripts: bash (with \ line continuations) and powershell (with ` line continuations).

Stick to one or the other.  And surround these commands with ```s.

Reply:
ok I will fix that.